Dockerfile

```diff
@@ -77,4 +77,10 @@ HEALTHCHECK --interval=30s --timeout=3s CMD curl --fail http://localhost:8080/pi
 
 COPY docker-init.sh /srv/init.sh
 RUN chmod +x /srv/init.sh
+COPY backend/app/cmd/testdata/cert.pem /cert.pem
```

Comment:
Is it related to battle.net oauth? Should it be present in production docker image? This doesn't seem right in this PR and at least deserves a comment in code (in Dockerfile).

Reply:
@paskal Nope, not related, these are leftovers from local testing. Removed